pkgs/applications/editors/vim/plugins/neovim-require-check-hook.sh

```diff
@@ -10,7 +10,7 @@ neovimRequireCheckHook () {
 		# editorconfig-checker-disable
         export HOME="$TMPDIR"
         @nvimBinary@ -es --headless -n -u NONE -i NONE --clean -V1 \
-            --cmd "set rtp+=$out,${dependencies/ /,}" \
```

Comment:
question: I'm unfamiliar with this syntax. Is it syntactic sugar for `builtins.concatStringsSep`?
If so, can we use `lib.concatStringsSep` instead?

Reply:
no, this is just bash
see the commit message: https://github.com/NixOS/nixpkgs/pull/342240/commits/b8581496dfb668d839e879c8e17393d44b726887